lib/embeds.js

```diff
@@ -11,63 +9,112 @@ function renderImage (props) {
     return '';
   }
 
-  return <amp-img width={width} height={height} layout='responsive' src={src}></amp-img>;
+  return <amp-img width={width} height={height} layout='responsive' src={src} />;
 }
 
-function renderYoutube (props) {
+renderImage.propTypes = {
+  width: React.PropTypes.number,
```

Comment:
@ellell sorry, actually, no. It should remove images that don't have width or height.